Jenkinsfile

```diff
@@ -0,0 +1,63 @@
+library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
+]) _
+
+pipeline {
+    agent {
+        label 'large-amazon'
+    }
+    parameters {
+        choice(
+            choices: [ 'test', 'e2e' ],
+            description: 'Run UI tests?',
+            name: 'RUN_TEST')
+        string(
+            defaultValue: '80.0',
+            description: 'Google Chrome version',
+            name: 'CHROME_VERSION')
+    }
+    stages {
+        stage('Prepare') {
+            steps {
+                installDocker()
+                withCredentials([usernamePassword(credentialsId: 'hub.docker.com', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
+                    sh """
+                        sg docker -c "
+                            docker login -u "${USER}" -p "${PASS}"
+                        "
+                    """
+                }
+                slackSend channel: '#pmm-ci', color: '#FFFF00', message: "[${JOB_NAME}]: build started - ${BUILD_URL}"
+            }
+        }
+        stage('UI tests') {
+            steps {
+                sh """
+                    sg docker -c "
+                        export CHROME_VERSION=${params.CHROME_VERSION}"
+                        make ${params.RUN_TEST}
+                    "
+                """
+                stash includes: 'results/docker/TAG', name: 'IMAGE'
+                archiveArtifacts 'results/docker/TAG'
+            }
+        }
+    }
+    post {
+        always {
+            script {
+                if (currentBuild.result == null || currentBuild.result == 'SUCCESS') {
+                    unstash 'IMAGE'
+                    def IMAGE = sh(returnStdout: true, script: "cat results/docker/TAG").trim()
+                    def CLIENT_IMAGE = sh(returnStdout: true, script: "cat results/docker/CLIENT_TAG").trim()
+                    slackSend channel: '#pmm-ci', color: '#00FF00', message: "[${JOB_NAME}]: build finished - ${IMAGE}"
```

Comment:
we can even tag users on channel with msg for test result, implemented the same with api tests